dxsp/utils/contract_utils.py

```diff
@@ -348,7 +348,9 @@ def __init__(
             self.block_explorer_api = block_explorer_api
             self.decimals = None
             self.name = None
-            logger.debug("{} - token initialized {}", self.symbol, self.address)
+            logger.debug(
+                "token initialized symbol {} address {}", self.symbol, self.address
+            )
         except Exception as error:
             logger.error("token error {}", error)
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consider including more context in the error log.

Adding additional context such as the token symbol or address might help in debugging issues more efficiently.

```suggestion
logger.error("token error {} - symbol: {}, address: {}", error, self.symbol, self.address)
```